depth_image_proc/include/depth_image_proc/conversions.hpp

```diff
@@ -51,9 +51,9 @@ namespace depth_image_proc
 template<typename T>
 void convertDepth(
   const sensor_msgs::msg::Image::ConstSharedPtr & depth_msg,
-  sensor_msgs::msg::PointCloud2::SharedPtr & cloud_msg,
+  const sensor_msgs::msg::PointCloud2::SharedPtr & cloud_msg,
   const image_geometry::PinholeCameraModel & model,
-  double range_max = 0.0)
+  double invalid_depth = 0.0)
```

Comment:
I actually think range_max is a better name simply because that's more in line with what a number of depth camera drivers define (as well as most laser range finders). I'd suggest we keep the name of this - but add a doxygen comment to the function to explain what range_max is used for.

Reply:
Did you check the implementation? If i Recall correctly ( i am currently writing from my phone) this parameter is only used for invalid points. So if there are Points with more than `range_max` distance (depth) they will not be clamped to this value, which would be indicated by its name IMHO